libkbfs/md_util.go

```diff
@@ -219,6 +221,84 @@ func getUnmergedMDUpdates(ctx context.Context, config Config, id TlfID,
 	return currHead, unmergedRmds, nil
 }
 
+func encryptMDPrivateData(
+	ctx context.Context, codec Codec, crypto cryptoPure,
+	signer cryptoSigner, ekg encryptionKeyGetter, me keybase1.UID,
+	rmd ReadOnlyRootMetadata, brmd *BareRootMetadata) error {
```

Comment:
I wanted to save a copy when we're building a `RMDSigned`, but that's not really worth it. Changed.